packages/cli/test/integration/generators/app.integration.js

```diff
@@ -74,3 +74,26 @@ describe('app-generator with --applicationName', () => {
     );
   });
 });
+
+// The test takes about 1 min to install dependencies
+describe.skip('app-generator with --format', () => {
```

Comment:
is the `skip` here intentional (I thought we wanted to disable on CI, but run locally)? Or do we want to enable it only when debugging it locally?